javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,94 @@
+//Item array removal :)
+
+const names = [
+    'Peter',
+    'Ahmad',
+    'Yana',
+    'kristina',
+    'Rasmus',
+    'Samuel',
+    'katrine',
+    'Tala',
+  ];
+  
+  names.splice(1, 1);
+  
+  console.log (names); 
+
+
+
+//When will we be there?
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  }
+  ;
+  function trip () {
+  
+    var time = travelInformation.destinationDistance/travelInformation.speed;
+    var hours = parseInt(Number(time));
+    var minutes = Math.round((Number(time)- hours) *60 );
+    console.log ("The total duration of the trip is: " + hours+ " hours and "+ minutes + " minutes.")
+  }
+  const travelTime = trip();
+  console.log(travelTime); 
+
+
+
+
+  //Series :D
+  const seriesDurations = [
+    {
+      title: "Community",
+      totalInMinutes: 2420,
+    },
+    {
+      title: "The office",
+      totalInMinutes: 4290,
+    },
+     {
+      title: "The Handmaid's tale",
+      totalInMinutes: 1800,
+    },
+  ];
+  
+  var life = (42048000);
```

Comment:
If I see the number 42048000, I do not know what it means. I can guess but it is better if you write code that tells me what it means. I think it is 80 years in minutes.
```suggestion
  var lifeInMinutes = (80 * 365 * 24 * 60);
```